app/models/response.rb

```diff
@@ -217,4 +217,83 @@ def self.get_volume_of_review_comments(assignment_id, reviewer_id)
     avg_vol_in_round_3 = (Lingua::EN::Readability.new(comments_in_round_3).num_words / (counter_in_round_3.zero? ? 1 : counter_in_round_3)).round(0)
     [overall_avg_vol, avg_vol_in_round_1, avg_vol_in_round_2, avg_vol_in_round_3]
   end
+  def notify_on_difference(new_pct, avg_pct, limit,response_id)
+    mapping = self.map
+    instructor = mapping.assignment.instructor
+    team_id = mapping.reviewee_id
+    teams_users = TeamsUser.where(team_id: team_id)
+    user_id=nil
+    teams_users.each do |team_user|
+      user_id=team_user.user_id;
```

Comment:
I don't understand the code here. You loop all the users in reviewee team, then what?